.github/workflows/ci.yml

```diff
@@ -663,3 +622,60 @@ jobs:
         with:
           working-directory: quic/s2n-quic-core
           args: --tests
+
+  dhat:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions-rs/toolchain@v1.0.7
+        id: toolchain
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - uses: camshaft/rust-cache@v1
+
+      - name: Run cargo build
+        working-directory: tools/memory-report
+        run: cargo build --release
+
+      - name: Run server
+        working-directory: tools/memory-report
+        run: ./target/release/memory-report server &
```

Comment:
What kills the server? Or is it just relying on GitHub killing everything when the workflow completes?

Reply:
yep GHA will clean it up

Reply:
https://github.com/aws/s2n-quic/blob/a7ea2095bdfae77047038719980ce0e97e201b71/.github/workflows/netbench.yml#L184